src/edu/wright/cs/raiderplanner/controller/MenuController.java

```diff
@@ -275,10 +285,30 @@ public void main() {
 			calendarOpen = false;
 			break;
 		}
-		//Based on user choice of menu option "Export Calendar" button is shown/hidden
+		// Based on user choice of menu option
+		// "Export Calendar" button is shown/hidden
 		exportCalBox.setVisible(calendarOpen);
 	}
 
+	/**
+	 * Returns a boolean value of whether or not there is a successful chat connection.
+	 *
+	 * @return the boolean value of whether or not there is a successful chat connection.
+	 */
+	public boolean getChatConnection() {
+		return chatConnection;
+	}
+
+	/**
+	 * Sets the boolean variable of whether or not there is a successful chat connection.
+	 *
+	 * @param newChatConnection
+	 *            - the value to which to set chatConnection
+	 */
+	public void setChatConnection(boolean newChatConnection) {
```

Comment:
The purpose of this setter method is not clear to me. Why would an outside class need to set the status of the chat connection? Is it not the responsibility of the controller to manage that?